### DISCUSSION TITLE
SSO auth

### DISCUSSION THREAD

**Discussion by sushilkhadka165** [saleor/saleor#7413] [Q&A]
Hello @patrys 

I couldn't find any documentation regarding the Saleor 3.0's external auth (SSO) endpoints. Could you please point me to the right direction so I could check and test it.

**Comment by krzysztofwolski** [comment]
Ready to use plugin is available in Saleor Cloud 
https://docs.saleor.io/docs/3.0/developer/available-plugins/openid-connect

Open source project has implemented interfaces which can be used for creating own SSO integration

**Comment by patrys** [comment]
The OIDC plugin is getting open-sourced: #9406
